main.py

```diff
@@ -742,8 +742,13 @@ async def command_form(message: Message) -> None:
     def escape_appscript(text: str) -> str:
         return text.replace("\\", "\\\\").replace('"', '\\"')
 
+    # Extract bust number from channel name
+    bust_number = "".join([c for c in current_channel.name if c.isdigit()])
+    if bust_number:
+        bust_number = bust_number + " "
+
     # Constants in generated code, Make sure these strings are properly escaped
-    default_title = "Busty's Voting"
+    default_title = f"Busty's {bust_number}Voting"
```

Comment:
Should this still be called `default_title` if it ends up also being the real title?

Reply:
Good point, I'll rename it `form_title`